documents/knowledge/elements.yaml

```diff
@@ -6345,6 +6345,27 @@
     - "[[]] : []"
     - "[[[], [], []]] : [[0, []], [1, []], [2, []]]"
 
+- element: "Þ‟"
+  name: List from Anti-Diagonals
+  description: List of lists from anti-diagonals of a list
+  arity: 1
+  overloads:
+    lst: List of lists from anti-diagonals of a list
+  vectorise: false
+  tests:
+    - "[[[1], [2, 6], [3, 7, 11], [4, 8, 12, 16], [5, 9, 13, 17, 21], [10, 14, 18, 22], [15, 19, 23], [20, 24], [25]]] : [[1, 2, 3, 4, 5], [6, 7, 8, 9, 10], [11, 12, 13, 14, 15], [16, 17, 18, 19, 20], [21, 22, 23, 24, 25]]"
```

Comment:
Maybe a test case for a non-square matrix?

Reply:
Pretty sure this only really applies to square matrixes